api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,1207 @@
+
+          The value set will influence, if not set otherwise, the default set of validation-strategies,
+          enrichment-strategies, and the effective_schema in the following way:
+
+          - `undefined`: No predefined changes apply. `effective_schema` is exactly the same as the
+            `EventTypeSchema`.
+            Default validation_strategy for this `EventType` is `[{name: 'schema-validation'}]`.
+
+          - `data`: Events of this type will be DataChangeEvents. `effective_schema` contains `metadata`, and
+            adds fields `data_op` and `data_type`. The passed EventTypeSchema defines the schema of `data`.
+            Default validation_strategy for this `EventType` is `[{name: 'datachange-schema-validation'}]`.
+
+          - `business`: Events of this type will be BusinessEvents. `effective_schema` contains `metadata`
+            and any additionally defined properties passed in the `EventTypeSchema` directly on top level of
+            the Event. If name conflicts arise, creation of this EventType will be rejected.
+            Default validation_strategy for this `EventType` is `[{name: 'schema-validation'}]`.
+      topic:
+        type: string
+        description: |
+          References the name of the low-level topic that is holding the stream of Events for this EventType.
+          Usually is the same as the name property.
+
+      effective_schema:
+        type: string
+        #read-only: true
+        description: |
+          The effective schema of this `EventType`. The predefined schema validator will use the value of
+          this field as base for the validation.
+          The generation of this value happens on creation of the `EventType` and is influenced by the
+          `EventType.type`.
+
+      validation_strategies:
+        description: |
+          Determines the validation that has to be executed upon reception of Events of this type. Events are
+          rejected if any of the rules fail (see details of Problem response on the Event publishing
+          methods).
+
+          Rule evaluation order is the same as in this array.
+
+          If not explicitly set, default value will respect the definition of the `EventType.type`.
+        type: array
+        items:
+          $ref: '#/definitions/EventValidationStrategy'
+
+      enrichment_strategies:
+        description: |
+          Determines the enrichment to be performed on an Event upon reception. Enrichment is performed once
+          upon reception (and after validation) of an Event and is only possible on fields that are not
+          defined on the incoming Event.
+
+          See documentation for the write operation for details on behaviour in case of unsuccessful
+          enrichment.
+        type: array
+        items:
+          $ref: '#/definitions/EventEnrichmentStrategy'
+        default: [{name: 'metadata-enrichment'}]
+
+
+      partition_resolution_strategy:
```

Comment:
editorial: 'strategy' is more vage and related to mid/long term activities. Would prefer 'mode' or something similar here. 
P.S: Also related to other usage of 'strategy' in the API.